conftest.py

```diff
@@ -183,6 +183,22 @@ def all_devices(request, device_params):
     ttnn.CloseDevices(devices)
 
 
+def reset_fabric(fabric_config):
+    import ttnn
+
+    # If fabric_config is not None, set it to FABRIC_2D
+    if fabric_config is not None:
```

Comment:
Will an empty `fabric_config` always be `None`?

Then you could do

```suggestion
    if fabric_config:
```

Reply:
Done